api/metrics-api-jaxrs-1.1/src/main/java/org/hawkular/metrics/api/jaxrs/handler/MetricHandler.java

```diff
@@ -82,6 +88,41 @@
     @HeaderParam(TENANT_HEADER_NAME)
     private String tenantId;
 
+    @POST
+    @Path("/")
+    @ApiOperation(value = "Create metric definition.", notes = "Clients are not required to explicitly create "
+            + "a metric before storing data. Doing so however allows clients to prevent naming collisions and to "
```

Comment:
In case I'm not very clear - it's late now :wink: - the metric definition is the data your provide to create a metric.